src/main/java/com/mozu/jobs/scheduler/JobScheduler.java

```diff
@@ -700,15 +701,28 @@ private Trigger createCronTrigger(String hours, String minutes, String timezone,
                 + ((siteId != null) ? "_" + siteId.toString() : "");
 
         TimeZone tz = TimeZone.getTimeZone(timezone);
-
+        
+        Integer[] randomNumbers=getRandomNumber();
```

Comment:
Just return a random integer here. And do the calc here.  Seems cleaner to me to have